src/util/color/predefinedcolor.h

```diff
@@ -0,0 +1,31 @@
+#ifndef PREDEFINEDCOLOR_H
+#define PREDEFINEDCOLOR_H
+
+#include <QColor>
+
```

Comment:
Missing: 
#include "util/memory.h"

Reply:
Is it needed if we are not using make_unique in this file?

Reply:
Yes, I cannot build it without. 
#include <memory> will work as well, but I might remember some issues with including it directly.

Reply:
Build just finished, I am curios to test:-)

Reply:
I'm adding the include (y)